package.json

```diff
@@ -7,13 +7,17 @@
     "email": "dom@hichroma.com"
   },
   "dependencies": {
-    "@storybook/design-system": "^5.1.10",
-    "core-js": "^3.6.5",
-    "gatsby": "^2.23.22",
+    "@mdx-js/mdx": "^1.6.16",
+    "@mdx-js/react": "^1.6.16",
+    "@storybook/design-system": "^5.1.11",
+    "core-js": "^2.6.11",
```

Comment:
Another thing that feels wrong, but was a time suck to figure out. Storybook was throwing all kinds of core-js errors if I didn't do this. Seems to be a recurring problem given:

https://github.com/storybookjs/frontpage/commit/5402a16b1faeefd654138350467f1df4a155d508#diff-b9cfc7f2cdf78a7f4b91a753d10865a2